kubernetes-client/src/com/goyeau/kubernetes/client/KubernetesClient.scala

```diff
@@ -12,39 +12,46 @@ import org.http4s.client.blaze.BlazeClientBuilder
 import org.http4s.client.jdkhttpclient.JdkWSClient
 
 import scala.concurrent.ExecutionContext
+import org.http4s.client.jdkhttpclient.WSClient
 
-case class KubernetesClient[F[_]: ConcurrentEffect: ContextShift](httpClient: Client[F], config: KubeConfig) {
-  lazy val namespaces = NamespacesApi(httpClient, config)
-  lazy val pods = PodsApi(
+case class KubernetesClient[F[_]: Sync](httpClient: Client[F], wsClient: WSClient[F], config: KubeConfig) {
```

Comment:
Is there any reason not to make `KubernetesClient` as class too instead of a case class?

Reply:
no, no reason. I will fix